finish/src/main/webapp/WEB-INF/web.xml

```diff
@@ -48,6 +48,7 @@
     </security-role>
 
     <!-- SECURITY CONSTRAINTS -->
+  <deny-uncovered-http-methods/>
```

Comment:
This does not align with the rest of the line?  Please check the spaces or tabs used.  I will move this to the bottom too (after the 2 constraints).